wagtail_modeltranslation/patch_wagtailadmin.py

```diff
@@ -141,7 +141,7 @@ def _patch_other_models(self, model):
             panels = filter(lambda field: field.field_name not in translation_registered_fields, panels)
             edit_handler = ObjectList(panels)
 
-            SNIPPET_EDIT_HANDLERS[model] = edit_handler.bind_to_model(model)
+            SNIPPET_EDIT_HANDLERS[model] = edit_handler.bind_to(model=model)
```

Comment:
Add a version check to keep compatibility for wagtail < 2.5

Reply:
@DiogoMarques29 I'm not familar with how to do that in this context, could you point me in the right direction?

Reply:
you could do something like:
```
from wagtail import VERSION
...
if VERSION < (2,5):
    SNIPPET_EDIT_HANDLERS[model] = edit_handler.bind_to_model(model)
else:
    SNIPPET_EDIT_HANDLERS[model] = edit_handler.bind_to(model=model)
```